packages/tabs/src/Tabs.ts

```diff
@@ -16,7 +16,7 @@ import {
     CSSResultArray,
     TemplateResult,
     PropertyValues,
-} from 'lit-element';
+} from '@spectrum-web-components/base';
```

Comment:
The margins in the `rtl` tabs look a little bit off: 

<img width="998" alt="Screen Shot 2020-08-03 at 1 27 45 PM" src="https://user-images.githubusercontent.com/8894489/89224500-6e8c1280-d58d-11ea-9698-5b1501331720.png">

We may need to update some of the overriding css in `tabs.css` to support `rtl`

Reply:
Good catch, great opportunity to update out processing so we don't need that manual override anymore!